src/main/scala/com/raquo/laminar/modifiers/EventListener.scala

```diff
@@ -29,6 +31,8 @@ class EventListener[Ev <: dom.Event, Out](
   @deprecated("Renamed: Use `domCallback` instead of `domValue`", "0.12.0")
   val domValue: js.Function1[Ev, Unit] = domCallback
 
+  val options: EventListenerOptions = new EventListenerOptions { capture = shouldUseCapture(eventProcessor); passive = shouldBePassive(eventProcessor) }
+
```

Comment:
Nice PR, thanks! Just a small formatting comment about this block:

- Please don't import anything under `dom.`, I always just import `dom` package itself, and use the `dom.` prefix in Laminar codebase to reduce confusion between native DOM types and Laminar types.

- Please don't use `;`, put these on separate lines.

Otherwise this is perfect. The change looks pretty straightforward, but did you have a chance to test it manually? While we do have a lot of tests for Laminar, they don't happen to cover event listener options.

Reply:
Thanks. I've made the change.

Regarding testing: I did manually confirm using the browser inspector that it would apply passive/capture when enabled and otherwise not:
<img width="314" alt="Screenshot 2023-04-01 at 15 03 21" src="https://user-images.githubusercontent.com/600700/229290683-c1984050-9879-4e9f-bb9d-3e721df09ba8.png">
<img width="315" alt="Screenshot 2023-04-01 at 15 06 59" src="https://user-images.githubusercontent.com/600700/229290815-b73b6f88-38ad-4222-9cf3-281ce1aa3fcd.png">
I'm not sure if there's a good way to test that programmatically.